lib/_http_outgoing.js

```diff
@@ -56,6 +57,7 @@ const HIGH_WATER_MARK = getDefaultHighWaterMark();
 const { CRLF, debug } = common;
 
 const kIsCorked = Symbol('isCorked');
+const kErrorEmitted = Symbol('errorEmitted');
```

Comment:
nit `Symbol('kErrorEmitted')`

Reply:
I've done it like the previous symbol `isCorked`.